### PR TITLE
Updated ModSDK files to playtest-20241116

### PR DESCRIPTION
**Pull request by penev92**
To get in line with https://github.com/OpenRA/OpenRAModSDK/pull/198.

The `.editorconfig` is a copy of the one used in the main OpenRA repo.
The rest is trivial.

### CODE REVIEW

**Comment by penev92** [comment]
Oops I didn't check for code violations after updating the `.editorconfig` 🤦‍♂️

**Comment by penev92** [comment]
Updated.

File: OpenRA.Mods.Mobius/Traits/World/RemasterTerrainRenderer.cs [hunk 1]
```diff
@@ -100,7 +100,7 @@ public void UpdateCell(CPos cell)
 		int t;
 		void ITick.Tick(Actor self)
 		{
-			t += 1;
+			t++;
```
Comment (pchote) [inline OpenRA.Mods.Mobius/Traits/World/RemasterTerrainRenderer.cs:103]: This could simplify further to `if (++t > 2)`

**Comment by pchote** [comment]
Can you please also change:

https://github.com/OpenRA/TiberianDawnHD/blob/56e4800c313f10c9f587d38c551235c8573199ad/make.ps1#L296

to:

```powershell
 $env:MOD_SEARCH_PATHS = "./mods" 
```

This should hopefully fix the Windows CI failure.

**Comment by penev92** [comment]
Updated.

**Pull request merged**